src/main/java/com/school/project/controller/GroupController.java

```diff
@@ -22,28 +22,37 @@
     private GroupService groupService;
 
     @PostMapping("/group")
-    public Group createGroup(@RequestBody GroupDto groupDto){
-        return groupService.create(convertGroupDtoToGroup(groupDto));
+    public ResponseEntity <Group> createGroup(@RequestBody GroupDto groupDto){
+        return ResponseEntity.ok()
+                .body(groupService.createUpdate(convertGroupDtoToGroup(groupDto)));
+    }
+
+    @PutMapping("/group/{groupId}/add/{userId}")
+    public void addUserToGroup(@PathVariable Long groupId, @PathVariable Long userId){
+        ResponseEntity.ok().body(convertGroupToGroupDto(groupService.addUser(groupId, userId)));
```

Comment:
why do you need body here? it should look like: line 32 - calling service, line 33 - returning Response